Doberman/utils.py

```diff
@@ -29,16 +29,22 @@ def find_plugin(name, path):
     :returns constructor: the constructor of the requested device
     """
     strip = False
+    split = False
     if not isinstance(path, (list, tuple)):
         path = [path]
     spec = importlib.machinery.PathFinder.find_spec(name, path)
     if spec is None:
         strip = True
         spec = importlib.machinery.PathFinder.find_spec(name.strip('0123456789'), path)
+    if spec is None:
+        split = True
+        spec = importlib.machinery.PathFinder.find_spec('_'.join(name.split('_')[:-1]), path)
```

Comment:
rsplit cleaner?

Reply:
ohhhh, I haven't seen that function! indeed, will change, thanks!